### PR TITLE
WIP: Inline Notice block

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

Trello task link - https://trello.com/c/I2JG5yiK

### Screenshots

https://share.getcloudapp.com/yAuY8kpW

### Types of changes

New feature (non-breaking change which adds functionality)

### How has this been tested?

Check all settings of a block in backend & front-end also.

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: assets/js/inline-notice.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+( function( $ ) {
```
Comment (RajkiranBagal) [inline assets/js/inline-notice.js:1]: Try to do this without jQuery

File: src/blocks/inline-notice/style.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
+
```
Comment (RajkiranBagal) [inline src/blocks/inline-notice/style.scss:1]: Make optimal use of SCSS
`.uagb-inline_notice__outer-wrap {
    &.uagb-inline_notice__align-right {
        text-align:right
    }
    &.uagb-inline_notice__align-center {
        text-align:center
    }
    &.uagb-inline_notice__align-left {
        text-align:left
    }

}`

File: assets/js/inline-notice.js (hunk 1 above)
Comment (AbhijitNage123) [reply assets/js/inline-notice.js]: Done

File: src/blocks/inline-notice/style.scss (hunk 2 above)
Comment (AbhijitNage123) [reply src/blocks/inline-notice/style.scss]: Done

Comment (RajkiranBagal) [reply src/blocks/inline-notice/style.scss]: Indent the code

File: src/blocks/inline-notice/edit.js [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+	AlignmentToolbar,
+	BlockControls,
+	InspectorControls,
+	RichText,
+	ColorPalette,
+} = wp.blockEditor
+
+const {
+	PanelBody,
+	SelectControl,
```
Comment (RajkiranBagal) [inline src/blocks/inline-notice/edit.js:32]: Remove unused declarations.

File: src/blocks/inline-notice/edit.js [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+			attributes: {
+				block_id,
+				icon,
+				noticeTitle,
+				noticeContent,
+				textColor,
+				titleColor,
+				noticeColor,
+				noticeDismissColor,
+				fontSize,
```
Comment (RajkiranBagal) [inline src/blocks/inline-notice/edit.js:71]: Remove unused declarations.

**Comment by RajkiranBagal** [comment]
@AbhijitNage123 There are multiple unused declarations in edit.js, save.js styling.js Please remove them.

File: src/blocks/inline-notice/save.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+	if ( icon ) {
+		image_icon_html = <span className="uagb-notice-dismiss">{ renderSVG(icon) }</span>
+	}
+
+
+	return (
+		<div className={ classnames(
+			className,
+			"uagb-inline_notice__outer-wrap",
```
Comment (RajkiranBagal) [inline src/blocks/inline-notice/save.js:40]: Indent the code properly - https://imgur.com/yFljgiG

**Comment by vrundakansara** [comment]
#### Add the below options:
- Ability to keep it Dismissable/Always Visible
- Vertical/Horizontal Padding for Content
- Vertical/Horizontal Padding for Title

#### Defaults
- Remove all default colors
- Keep background color default - #FFB901
- Notice Display - Always allow.

File: src/blocks/inline-notice/edit.js (hunk 4 above)
Comment (AbhijitNage123) [reply src/blocks/inline-notice/edit.js]: Done

File: src/blocks/inline-notice/edit.js (hunk 3 above)
Comment (AbhijitNage123) [reply src/blocks/inline-notice/edit.js]: Done

File: assets/js/inline-notice.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+	var notices = document.querySelectorAll(
+		'.wp-block-uagb-inline-notice.uagb-inline_notice__outer-wrap[data-id]'
+	);
+
+	notices.forEach( function( element ) {
+		var unique_id = element.getAttribute( 'data-id' );
+
+		var dismissible = element.querySelector( '.uagb-notice-dismiss' );
+
+		console.log(localStorage);
```
Comment (RajkiranBagal) [inline assets/js/inline-notice.js:11]: Remove console.log

Comment (AbhijitNage123) [reply assets/js/inline-notice.js]: Done.

**Comment by RajkiranBagal** [comment]
@AbhijitNage123 For the dismissible notice when I enable cookies and set for 1 day and then close the notice and refresh the page then the Notice is visible for 2 seconds and then disappears.

Screencast - https://share.getcloudapp.com/geuwYzkb

**Pull request merged**